src/screens/TopicPageScreen/index.js

```diff
@@ -422,6 +430,10 @@ const TopicPageScreen = (props) => {
         snap
         contentOffset={{x: 0, y: topPosition}}
         contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
+        onMomentumScrollEnd={(event) => {
+          onWillSendViewPostTime(event, feeds);
+          fetchNextFeeds(event);
+        }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onMomentumScrollEnd` event handler now calls two functions: `onWillSendViewPostTime` and `fetchNextFeeds`. This is a good way to handle multiple actions on the same event. However, it's important to ensure that these functions don't throw any unhandled exceptions that could cause the event handler to fail. Consider wrapping these calls in a try-catch block.

```diff
- onMomentumScrollEnd={(event) => {
-   onWillSendViewPostTime(event, feeds);
-   fetchNextFeeds(event);
- }}
+ onMomentumScrollEnd={async (event) => {
+   try {
+     await onWillSendViewPostTime(event, feeds);
+     await fetchNextFeeds(event);
+   } catch (error) {
+     console.error('Error handling scroll end event:', error);
+   }
+ }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->